src/Meilisearch/Index.cs

```diff
@@ -15,7 +15,7 @@ namespace Meilisearch
     public class Index
     {
         private HttpClient http;
-        private TaskEndpoint _taskEndpoint;
+        private TaskEndpoint taskEndpoint;
```

Comment:
I solved these issues some PR's before, can you update your main branch with our main?

https://github.com/meilisearch/meilisearch-dotnet/blob/main/src/Meilisearch/Index.cs#L18

Reply:
Sorry for this one...